test/cloudtest/pkg/commands/main.go

```diff
@@ -1184,6 +1190,8 @@ func ExecuteCloudTest() {
 
 func initCmd(rootCmd *cloudTestCmd) {
 	cobra.OnInitialize(initConfig)
+	rootCmd.Flags().StringVarP(&rootCmd.cmdArguments.currentBranch, "currentBranch", "", "", "Current running branch")
```

Comment:
I do not think it is usefull, please only pass CURRENT, MASTER Branch using ENV variables in configuration to tests, and check inside function to make logs.

Reply:
why tests should know about what the branch they running on?